public/components/common/seach/search.tsx

```diff
@@ -77,7 +78,24 @@ function Search (props: any) {
             liveStreamChecked={props.liveStreamChecked}
             onLiveStreamChange={props.onLiveStreamChange}
           />
-          <EuiFlexItem
+          { actionItems.length > 0 && (
+            actionItems.map((item) => {
+              return (
+                <EuiFlexItem
+                  className={ item.className ? item.className : "euiFlexItem--flexGrowZero"}
+                >
+                  <EuiButton 
+                    iconType={ item.iconType }
+                    { ...item.attributes }
+                    { ...item.handlers }
+                  >
+                    { item.text }
+                  </EuiButton>
+                </EuiFlexItem>
+              );
+            })
+          ) }
+          {/* <EuiFlexItem
```

Comment:
Do we have a future plan or `TODO` for this commented code snippet? If not, is it safe to remove?

Reply:
No, this one is supposed to be removed, i'll remove it.